src/test/codegen/no-jump-tables.rs

```diff
@@ -0,0 +1,17 @@
+// Test that the `no-jump-tables` function attribute are (not) emitted when
+// the `-Zno-jump-tables` flag is (not) set.
+
+// revisions: unset set
+// needs-llvm-components: x86
+// compile-flags: --target x86_64-unknown-linux-gnu
```

Comment:
Needs to be a no_core test for this to work.

Reply:
Ah, that explains why other tests did it. Thanks!